docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,444 @@
+  val outgoing = IO(new MixedAlignmentBundle)
+  incoming.flippedChild := DontCare // Otherwise FIRRTL throws an uninitialization error
+  outgoing :<= incoming
+}
+```
+
+This generates the following Verilog, where the aligned members are driven `incoming` to `outgoing` and flipped members are ignored:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example2())
+```
+
+### Flipped connection operator (:>=)
+
+For connections where you want the flipped-half of 'bulk-connect-like-semantics' where the aligned members are ignored and flipped members are assigned consumer-to-producer, use `:<=` (the "flipped connection", or "backpressure connection").
+
+```scala mdoc:silent
+class Example3 extends RawModule {
+  val incoming = IO(Flipped(new MixedAlignmentBundle))
+  val outgoing = IO(new MixedAlignmentBundle)
+  outgoing.alignedChild := DontCare // Otherwise FIRRTL throws an uninitialization error
+  outgoing :>= incoming
+}
+```
+
+This generates the following Verilog, where the aligned members are ignore and the flipped members are driven `outgoing` to `incoming`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example3())
+```
+
+> Note: Astute observers will realize that semantically `c :<>= p` is exactly equivalent to `c :<= p` followed by `c :>= p`.
+
+### Coercing mono-direction connection operator (:#=)
+
+For connections where you want to every producer member to always drive every consumer member, regardless of alignment, use `:#=` (the "coercion connection"). This operator is useful for initializing wires whose types contain members of mixed alignment.
+
+```scala mdoc:silent
+import chisel3.experimental.BundleLiterals._
+class Example4 extends RawModule {
+  val w = Wire(new MixedAlignmentBundle)
+  dontTouch(w) // So we see it in the output verilog
+  w :#= (new MixedAlignmentBundle).Lit(_.alignedChild -> true.B, _.flippedChild -> true.B)
+}
+```
+
+This generates the following Verilog, where all members are driven from the literal to `w`, regardless of alignment:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example4())
+```
+
+> Note: Astute observers will realize that semantically `c :#= p` is exactly equivalent to `c :<= p` followed by `p :>= c` (note `p` and `c` switched places in the second connection).
+
+## Waived Data
+
+It is not uncommon for a user to want to connect Chisel components which are not type equivalent. For example, a user may want to cook up the `ready`/`valid` members of a `ReadyValidIO` to a `DecoupledIO`, but
+because the `bits` member is not present in both, our operators would reject a connection.
+
+`WaivedData` is the mechanism to specialize connection operator behavior in these scenarios. For any addition member which is not present in the other component being connected to, they can be explicitly waived from the operator to be ignored, rather than trigger an error.
```

Comment:
I don't think WaivedData is the right thing in this example. because there is a superclassing relationship between DecoupledIO and ReadyValidIO, first of all you have to to a scala type-cast as well as the waivable. I think .viewAsSuperclass is the right pattern to suggest here.